allennlp/commands/elmo.py

```diff
@@ -11,8 +11,6 @@
 For information, see "Deep contextualized word representations", Peters et al 2018.
 https://arxiv.org/abs/1802.05365
 
-.. code-block:: console
```

Comment:
Did you check that these don't need to be replaced by triple backticks?

Reply:
Partially.  I'm not done with this PR.  For some reason part of it formats correctly and part doesn't.

Reply:
oh, sorry. To be clear I think these do need bacticks, as they are the output of a terminal help output.

Reply:
Indentation will also format a code block, but what I had didn't quite work because the command example sometimes only had three spaces but four are required to trigger a code block.  I probably should have just used backticks ;-)